ivlsim/Makefrag-iverilog

```diff
@@ -0,0 +1,174 @@
+#--------------------------------------------------------------------
+# Verilog Generation
+#--------------------------------------------------------------------
```

Comment:
will this install ivlsim? Or can ivlsim be installed with a package manager? If you want to add this you should show it working within the Travis regression

Reply:
(You can do this by editing the `.travis.yml` file in this repo)

Reply:
Hi, mwachs5

I am newbie of Travis, I try the original .travis.yml on my travis
https://travis-ci.org/jimmysitu/rocket-chip/builds/367613403
It seems I cannot complete **make tools** in 50mins and Travis timeout.( job #1.1)
Then I check the travis log of the PR, this step done less than 3mins, I think that the travis cache helps.
My question is if there is any way to make my rocket-chip .travis.yml works without timeout?
Thanks.